pkg/alertmanager/validation.go

```diff
@@ -197,10 +242,13 @@ func validatePushoverConfigs(configs []monitoringv1alpha1.PushoverConfig) error
 		}
 
 		if config.Retry != "" {
-			_, err := time.ParseDuration(config.Retry)
+			d, err := time.ParseDuration(config.Retry)
 			if err != nil {
 				return errors.New("invalid retry duration")
 			}
+			if d < 30*time.Second {
```

Comment:
i missed the reference for this check

Reply:
```yaml
# How often the Pushover servers will send the same notification to the user.
# Must be at least 30 seconds.
[ retry: <duration> | default = 1m ]
```

Reply:
ack but I don't think that Alertmanager would enforce the limit.

Reply:
ok fair enough, in that case then I think we can remove the duration checks entirely in this block since they are covered now with OpenApi validation?

Reply:
sgtm